tests/data/usr/bin/bootctl

```diff
@@ -0,0 +1,20 @@
+#! /bin/sh
```

Comment:
should a mock for sdbootutil also be installed?

Reply:
There is one.